src/utils/bytes.ts

```diff
@@ -0,0 +1,5 @@
+const BYTES_32_LENGTH = 66;
+
+export function isBytes32(value: string) {
+  return value && value.length === BYTES_32_LENGTH;

```

Comment:
i thought about this but i'm not sure if it would be correct because of examples like this one i saw on richContracts: 

![image](https://user-images.githubusercontent.com/5303411/202852933-750d243a-54c2-4635-ab5b-a7fdae62d56e.png)

this ones are component text so it isn't exactly the same but i thought it could happen. 

just to be clear, if there is a possibility of having on richContracts a component "string" and type "bytes32" that it doesn't start with 0x we shouldn't add this condition. 

Let me know what to think, no problem on adding it!

Reply:
We are using the text input here because bytes 32 accepts numbers and text, so the bytes32 is just verification on the text. So we don't change anything about how we pass the data, just if we accept it. So since all bytes will have 0x at the start we can have 0x check as well as length (coming from the 32 number)
But we can merge this now and add it later